app/helpers/config/aca_helper.rb

```diff
@@ -153,6 +153,26 @@ def fetch_plan_title_for_single_plan
     Settings.plan_option_titles.single_plan
   end
 
+  def fetch_health_product_option_choice_description_for_sole_source
+    Settings.plan_option_descriptions.sole_source
+  end
+
+  def fetch_health_product_option_choice_description_for_metal_level
+    Settings.plan_option_descriptions.metal_level
+  end
+
+  def fetch_health_product_option_choice_description_for_single_carrier
+    Settings.plan_option_descriptions.single_carrier
+  end
+
+  def fetch_health_product_option_choice_description_for_single_plan
+    Settings.plan_option_descriptions.single_plan
+  end
+
+  def fetch_dental_product_option_choice_description_for_single_plan
+    Settings.plan_option_descriptions.dental.single_plan
+  end
+
```

Comment:
1.Fetching all plan option description from settings file
2. Added new methods for each plan option.